qcengine/programs/empirical_dispersion_resources.py

```diff
@@ -662,6 +662,21 @@
             "hf": {"params": {"s6": 1.000, "s8": 0.713190, "a1": 0.079541, "a2": 3.627854}},  # JBS 01/2021
         },
     },
+    "d3op": {
+        "formal": "D3(op)",
```

Comment:
```suggestion
        "formal": "D3(OP)",
```
can this be all-caps for consistency? I suspect psi4 is expecting it so.

Reply:
It was called D3(op) with lowercase *op* in the original publication: https://pubs.acs.org/doi/10.1021/acs.jctc.7b00176.